test/utility/address.ts

```diff
@@ -0,0 +1,16 @@
+import { invokeTestCli } from '.'
+import { Addresses } from '../../src/command/addresses'
+
+export async function getPssAddress(beeDebugApiUrl: string): Promise<string> {
+  const execution = await invokeTestCli(['addresses', '--bee-debug-api-url', beeDebugApiUrl])
+
+  return (execution.runnable as Addresses).nodeAddresses.pssPublicKey
+}
+
+export function getWorkerPssAddress(stringLength: number): string {
```

Comment:
Cool, it would be good to have this is bee-js too :)
https://github.com/ethersphere/bee-js/issues/250